packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -43,6 +43,12 @@
   "force_source_status_local_text": "SFDX: View Local Changes",
   "force_source_status_remote_text": "SFDX: View Changes in Default Org",
   "force_source_status_text": "SFDX: View All Changes (Local and in Default Org)",
+  "cancel_sfdx_command_text": "SFDX: Cancel Active Command",
+  "apex_generate_class_text": "SFDX: Create Apex Class",
```

Comment:
@WillieRuemmele this key already exists above, please reconcile these additions to see if they need to be added.